consensus/consensus-types/src/block_retrieval.rs

```diff
@@ -71,6 +83,55 @@ impl fmt::Display for BlockRetrievalRequest {
     }
 }
 
+/// RPC to get a chain of block of the given length starting from the given block id.
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq)]
+pub struct BlockRetrievalRequestV2 {
+    block_id: HashValue,
+    num_blocks: u64,
+    // TODO: remove the Option, if it's not too painful
```

Comment:
Ah, two things I now remember thinking to clean up:
1. Now that we're separating out the logic, can we remove the Option? It doesn't make sense to have a V2 request without the target
2. Is epoch needed here?

Reply:
1. Sounds good. For posterity in case we come back to this, the option and `(epoch, round)` was included from the original [PR](https://github.com/aptos-labs/aptos-core/pull/14076/files#diff-ec09b6e8b45ed2757da9b3cba4d363ee9cc972240ae28d019db0ea9fc5e9f4c4R158-R163)
2. Will remove epoch and Option